src/JJMasterData.Commons/Extensions/StringExtensions.cs

```diff
@@ -0,0 +1,43 @@
+﻿using JJMasterData.Commons.Util;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+using System.Linq;
+
+namespace JJMasterData.Commons.Extensions
+{
+    public static class StringExtensions
+    {
+        /// <summary>
+        /// Remove acentos da string
+        /// </summary>
+        /// <param name="text"></param>
+        /// <returns></returns>
+        public static string RemoveAccents(this string text)
```

Comment:
This method already exists in StringManager.NoAccents.
Please avoid Extensions methods to string